### PR TITLE
G1 2024 v3 #14685

### PR DESCRIPTION
**Pull request by a22ricni**
Made it so the label should not be moving anymore. Can test it by have two entities close to each other and have a line between them. After set a label on the line and select on of the entity and move it by the arrow keys. It also depends on what zoom you have on the camera.

### CODE REVIEW

**Review by a22wilas — APPROVED** [review]
Old version:
bild

This branch:
bild

From testing I can confirm that the label bug happens when the entity elements are verticaly aligned and also depends on the campera position. If the snap to grind function is on, the label sometimes stays in place. 
On the new branch the label always stays in place, both if snap to grid is on or off. Tried this on multiple elements with different camera positions. I approve that the change has fixed the bug!

TLDR
bild
For some reason in this branch the icons have increased in size, this does not seem to have anything to do with the code changed in this issue. I changed the code back to it's original code from the issue but did not effect the size of the icons in the toolbar.
